include/stronk/skills/can_hash.hpp

```diff
@@ -1,22 +1,23 @@
 #pragma once
-#include <concepts>
 #include <cstddef>
 #include <functional>
 #include <type_traits>
 
+#include "stronk/stronk.hpp"
+
 namespace twig
 {
 
+// Deprecated, no need to add as a skill anymore
```

Comment:
Can we use this to make tooling point it out? https://en.cppreference.com/w/cpp/language/attributes/deprecated

Reply:
Im sure I looked up it didnt work on structs - but it looks like it does...